server/src/routes/api/v1/showsRouter.js

```diff
@@ -2,13 +2,16 @@ import express from "express"
 import objection from "objection"
 const { ValidationError } = objection
 import { Show } from "../../../models/index.js"
+import { Review } from "../../../models/index.js"
+import showsReviewRouter from "./showsReviewsRouter.js"
 import ShowSerializer from "../../../serializers/ShowSerializer.js"
 import showsReviewsRouter from "./showsReviewsRouter.js"
 import cleanUserInput from "../../../services/cleanUserInput.js"
 
 const showsRouter = new express.Router()
 
 showsRouter.use("/:id/reviews", showsReviewsRouter)
+showsRouter.use("/shows/:id", showsReviewRouter)
```

Comment:
this looks like you're expecting a path to be:
```js
/api/v1/shows/shows/:id
```
If we're already in the showsRouter, any endpoint starting with "api/v1/shows" will be sent here, then it's tacking on another `/shows` in the route?